src/hooks/useDailyPriceStats.ts

```diff
@@ -22,6 +22,37 @@ export const TOKEN_LIST = [
   Token.Bonk,
 ];
 
+export function getTokenGivenAddress(address: string) {
+  switch (address) {
+    case "So11111111111111111111111111111111111111112":
+      return Token.SOL;
+    case "4zMMC9srt5Ri5X14GAgXhaHii3GnPAEERYPJgZJDncDU":
+      return Token.USDC;
+  }
+}
+
+// function that maps token string to token enum
+export function getTokenGivenString(token: string) {
```

Comment:
it looks like this function is meant to be a typeguard of sorts to cast a string as a `Token`. In that case, it might be better to name this function `asToken`. And, you will also want to add a default case at the end of the switch statement that throws. Something like `default: throw new Error('not a valid token')`.